content/doc/book/managing/plugins.adoc

```diff
@@ -85,6 +85,9 @@ configured Update Center.
 [[install-with-cli]]
 === Using the Jenkins CLI
 
+.Installing Jenkins Plugins using Jenkins CLI
```

Comment:
I think your replacement text `{JOSS}` is referencing an asciidoc attribute that is not available in the Jenkins documentation source code.

I would rather not provide that attribute since the name "Jenkins" is a [registered trademark](https://www.jenkins.io/project/trademark/) in the USA.